### PR TITLE
feat(brand-icons): adds brand icon for currency pounds

### PR DESCRIPTION
**Pull request by HarshaDewani**
# Description

Adds a brand icon for currency pounds.

Storybook link: (once netlify has deployed link provide a link to the component)
Design link: https://legalandgeneral.invisionapp.com/console/share/JKZ6TDGSC4G/829959287/inspect
Screenshot: 
image

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)

### CODE REVIEW

File: projects/canopy/src/lib/brand-icon/brand-icons.interface.ts [hunk 1]
```diff
@@ -4,161 +4,168 @@ export const lgBrandIconCalendar: {
...
...-19.84c0-2.296 1.792-4.159 4-4.159zm36 96H24v183.675c0 .156.109.288.253.318l.064.007H295.68a.321.321 0 0 0 .313-.26l.006-.065V112zm-40 24c.754 0 1.46.222 2.062.608 1.164.707 1.938 1.962 1.938 3.392l-.002.11.002.14v127.497l-.002.142.002.111a4 4 0 0 1-4 4H64c-.76 0-1.472-.226-2.077-.618-1.156-.71-1.923-1.96-1.923-3.382l.002-.108-.001-.072L60 140.25l.001-.142A3.945 3.945 0 0 1 60 140c0-1.42.764-2.668 1.916-3.378A3.805 3.805 0 0 1 64 136h192zm-132 72H68v56h56v-56zm64 0h-56v56h56v-56zm64 0h-56v56h56v-56zm-12 40a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm-32 0a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm16-16a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm16-16a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm-32 0a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm-84-72H68v56h56v-56zm64 0h-56v56h56v-56zm64 0h-56v56h56v-56zm43.68-96L264 47.999v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48H64v19.842C64 70.137 62.208 72 60 72s-4-1.863-4-4.159V48H24.317a.32.32 0 0 0-.31.26l-.007.065V104h272V48.325a.323.323 0 0 0-.32-.325z"/></g></svg>`,
```
Comment (HarshaDewani) [inline projects/canopy/src/lib/brand-icon/brand-icons.interface.ts:7]: Not sure why it shows here that something is changed 🤔

Comment (owensgit) [reply projects/canopy/src/lib/brand-icon/brand-icons.interface.ts:7]: Try doing an `npm i`, we recently updated the packages which create and optimise these interfaces, so you may have a slightly older version.

**Comment by owensgit** [comment]
Thanks for adding this Harsha, could you also add the brand icon SVG file to **_projects/canopy/src/assets/brand-icons_**?

File: projects/canopy/src/lib/brand-icon/brand-icons.interface.ts (hunk 1 above)
Comment (HarshaDewani) [reply projects/canopy/src/lib/brand-icon/brand-icons.interface.ts]: Done :) 
Thanks!!

**Comment by HarshaDewani** [comment]
> Thanks for adding this Harsha, could you also add the brand icon SVG file to **_projects/canopy/src/assets/brand-icons_**?

Updated

**Review by elenagarrone — APPROVED** [review]

**Review by owensgit — APPROVED** [review]

**Pull request merged**